src/Transform/ONNX/ConstProp.cpp

```diff
@@ -63,10 +63,14 @@ namespace {
 struct ConstPropONNXToONNXPassConfiguration {
   static int expansionBound;
   static StringSet<> disabledPatterns;
+  static int optLevel;
+  static bool constantPropIsEnabled;
 };
 
 int ConstPropONNXToONNXPassConfiguration::expansionBound = -1; // -1 == no bound
 StringSet<> ConstPropONNXToONNXPassConfiguration::disabledPatterns;
+int ConstPropONNXToONNXPassConfiguration::optLevel;
+bool ConstPropONNXToONNXPassConfiguration::constantPropIsEnabled;
```

Comment:
please assign explicit defaults to these, like expansionBound, so it's clear what happens if configureConstPropONNXToONNXPass() isn't called

Reply:
okay I can update it